CONTRIBUTING.md

```diff
@@ -255,143 +285,83 @@ cuda-memcheck python <program_name>.py <program_arguments>
 
 ### Device debug symbols
 
-The device debug symbols are not automatically added with the cmake `Debug`
-build type because it causes a runtime delay of several minutes when loading
-the libcudf.so library.
+The device debug symbols are not automatically added with the cmake `Debug` build type because it
+causes a runtime delay of several minutes when loading the libcudf.so library.
 
-Therefore, it is recommended to add device debug symbols only to specific files by
-setting the `-G` compile option locally in your `cpp/CMakeLists.txt` for that file.
-Here is an example of adding the `-G` option to the compile command for
-`src/copying/copy.cu` source file:
+Therefore, it is recommended to add device debug symbols only to specific files by setting the `-G`
+compile option locally in your `cpp/CMakeLists.txt` for that file. Here is an example of adding the
+`-G` option to the compile command for `src/copying/copy.cu` source file:
 
 ```
 set_source_files_properties(src/copying/copy.cu PROPERTIES COMPILE_OPTIONS "-G")
 ```
 
-This will add the device debug symbols for this object file in libcudf.so.
-You can then use `cuda-dbg` to debug into the kernels in that source file.
-
-### Building and Testing on a gpuCI image locally
-
-Before submitting a pull request, you can do a local build and test on your machine that mimics our gpuCI environment using the `ci/local/build.sh` script.
-For detailed information on usage of this script, see [here](ci/local/README.md).
-
+This will add the device debug symbols for this object file in `libcudf.so`.  You can then use
+`cuda-dbg` to debug into the kernels in that source file.
 
-## Automated Build in Docker Container
+## Code Formatting
 
-A Dockerfile is provided with a preconfigured conda environment for building and installing cuDF from source based off of the main branch.
+### Python
 
-### Prerequisites
+cuDF uses [Black](https://black.readthedocs.io/en/stable/), [isort](https://pycqa.github.io/isort/),
+and [flake8](https://flake8.pycqa.org/en/latest/) to ensure a consistent code format throughout the
```

Comment:
Also `mypy`?

Reply:
Alternatively, I considered eliminating this entire section about black/isort/flake8 in favor of just describing how to use pre-commit. That is guaranteed to be up-to-date since it's exactly how we run these tools in CI. What are your thoughts on that?

Reply:
Yeah, I think something along the lines of [this](https://docs.dask.org/en/latest/develop.html#code-formatting) would be great

Reply:
Love it. I'll adopt similar wording here.

Reply:
Done in 039fc0640e.